src/DataMapper/Product/Provider/ProductPrices.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\DataMapper\Product\Provider;
+
+final class ProductPrices
+{
+    public function __construct(
+        public readonly ?int $price = null,
+        public readonly ?int $originalPrice = null,
```

Comment:
It really doesn't matter, but a value object like this shouldn't have both properties nullable imo. But again, it doesn't matter in our use case